README.md

```diff
@@ -71,6 +71,16 @@ module.exports = {
         return path.indexOf(`${componentMeta.fileNameWithoutPrefix}.spec`) !== -1;
     },
 
+    /**
+     * This function will wrap the content of sandboxes, 
+     * for example to use special context providers within provided
+     * Class or Component
```

Comment:
The description is not clear enough and is misleading. Does this function wrap anything on its own?
Can we add something in addition to contexts?